nilearn/tests/test_signal.py

```diff
@@ -329,7 +328,7 @@ def test_clean_confounds():
 def test_high_variance_confounds():
     # C and F order might take different paths in the function. Check that the
     # result is identical.
-    n_features = 1001
+    n_features = 1000
```

Comment:
I seem to remember this was related to removing one warning so this change and the one above can be removed